Outsourcing Environment Variables to the Documentation (#1948)
docker-mailserver/docker-mailserver: docs/content/contributing/issues-and-pull-requests.md

3. Add integration tests if necessary
4. Get the linters with `make install_linters` and install `jq` with the package manager of your OS
5. Use `make clean all` to build image locally and run tests (note that tests work on Linux **only**)

> 6. Document your improvements if necessary (e.g. if you introduced new environment variables, write the description on [`this page`][docs-environment])

Reviewer: ```suggestion
6. Document your improvements if necessary (e.g. if you introduced new environment variables, describe those in the [ENV documentation][docs-environment])
```
